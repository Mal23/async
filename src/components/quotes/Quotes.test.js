import React from 'react';
import { shallow } from 'enzyme';
import Quotes from './Quotes';

describe('Quotes component', () => {
  it('renders Quotes', () => {
    const quotes = [
      {
        character: 'Fry',
        quote: 'What??',
        image: '/image'
      },
      {
        character: 'Fry',
        quote: 'Huh??',
        image: '/image'
      }
    ];
    const wrapper = shallow(<Quotes quotes={quotes} />);
    expect(wrapper).toMatchSnapshot();
  });
});
