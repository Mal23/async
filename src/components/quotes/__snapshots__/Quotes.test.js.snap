// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes component renders Quotes 1`] = `
<ul
  className="Quotes"
>
  <li
    key="0"
  >
    <Quote
      quote={
        Object {
          "character": "Fry",
          "image": "/image",
          "quote": "What??",
        }
      }
    />
  </li>
  <li
    key="1"
  >
    <Quote
      quote={
        Object {
          "character": "Fry",
          "image": "/image",
          "quote": "Huh??",
        }
      }
    />
  </li>
</ul>
`;
